experiments/AtmosGCM/GCMDriver/gcm_bcs.jl

```diff
@@ -0,0 +1,77 @@
+# Boundary conditions
+
+# parse `--surface-flux` command line argument
+function parse_surface_flux_arg(arg, FT, param_set, orientation, moisture)
+    if arg === nothing || arg == "default"
+        boundarycondition = (AtmosBC(), AtmosBC())
+    elseif arg == "bulk"
+        if !isa(moisture, EquilMoist)
+            error("need a moisture model for surface-flux: bulk")
+        end
+        _C_drag = C_drag(param_set)::FT
+        bulk_flux = Varying_SST_TJ16(param_set, orientation, moisture)
+        boundarycondition = (
+            AtmosBC(
+                energy = BulkFormulaEnergy(
+                    (state, aux, t, normPu_int) -> _C_drag,
+                    (state, aux, t) -> bulk_flux(state, aux, t),
+                ),
+                moisture = BulkFormulaMoisture(
+                    (state, aux, t, normPu_int) -> _C_drag,
+                    (state, aux, t) -> begin
+                        _, q_tot = bulk_flux(state, aux, t)
+                        q_tot
+                    end,
+                ),
+            ),
+            AtmosBC(),
+        )
+    else
+        error("unknown surface flux: " * arg)
+    end
+
+    return boundarycondition
+end
+
+"""
+Defines analytical function for prescribed T_sfc and q_sfc, following
+Thatcher and Jablonowski (2016), used to calculate bulk surface fluxes.
+T_sfc_pole = SST at the poles (default: 271 K), specified above
+"""
+struct Varying_SST_TJ16{PS, O, MM}
+    param_set::PS
+    orientation::O
+    moisture::MM
+end
+function (st::Varying_SST_TJ16)(state, aux, t)
+    FT = eltype(state)
+    φ = latitude(st.orientation, aux)
```

Comment:
`st` doesn't seem to be defined in this function

Reply:
`st` is the identifier for the type -- see line 46. This is a method definition for a callable type.